.github/workflows/analysis.yml

```diff
@@ -141,6 +142,9 @@ jobs:
       - name: Install C++ compiler and libraries
         run:  sudo apt-get install $(cat .github/packages/ubuntu-20.04-apt.txt)
 
+      - name: Install dependencies
+        run:  sudo pip3 install --upgrade meson ninja
+
```

Comment:
(minor) might as well combined these into "Install dependencies", for consistency w/ the debug build block.

Reply:
In commit message, suggest mentioning that Meson 54.2 is required to fix bugs in compiler built-in detection (which isn't provided by the outdated apt-repo version).

Reply:
I haven't actually tested which version of Meson fixes the built-in logic.

Reply:
> Suggest mentioning in the commit comment that Meson >= 54.2 is required to fix bugs in its compiler built-in detection (which, isn't currently provided in Ubuntu 20.04's apt repo package).

Should we knock `meson` out of `ubuntu-20.04-apt.txt`?

Reply:
Good call - I think it makes sense to as of now. 

Oh, let's also bump the minimum required Meson version to 54.2 at the top of `meson.build`.